server/main.py

```diff
@@ -0,0 +1,138 @@
+import shutil
+from fastapi import FastAPI, HTTPException, Form
+from fastapi.responses import JSONResponse
+from fastapi.middleware.cors import CORSMiddleware
+from typing import Optional, List
+from server.app.crud.crud import check_key_existence,create_connection,create_image_metadata, get_metadata, delete_metadata,db_file
+import os
+import string
+import random
+from fastapi import HTTPException
+app = FastAPI()
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+connection=create_connection(db_file)
+
+def generate_random_string(length=8):
+    letters = string.ascii_lowercase
+    return ''.join(random.choice(letters) for _ in range(length))
+
+@app.post("/update_file/")
```

Comment:
This should be upload. Why did you change the path itself?
Look at the documentation, API -spec says /upload and the function name should be upload. Why did you change it to update?